core/lib/Cardano/Address.hs

```diff
@@ -237,3 +220,15 @@ invariantNetworkTag limit (NetworkTag num)
       ++ show num
       ++ ", but expected to be less than "
       ++ show limit
+
+newtype AddressVerificationKeyHash = AddressVerificationKeyHash { unAddressVerificationKeyHash :: ByteString }
```

Comment:
```suggestion
newtype AddressVerificationKeyHash =
    AddressVerificationKeyHash { unAddressVerificationKeyHash :: ByteString }
```

https://github.com/input-output-hk/adrestia/wiki/Coding-Standards#limit-line-length-to-80-characters